src/main/resources/sodium.mixins.json

```diff
@@ -38,6 +38,7 @@
     "features.entity.smooth_lighting.MixinEntityRenderer",
     "features.fast_biome_colors.MixinBackgroundRenderer",
     "features.fast_biome_colors.MixinClientWorld",
+    "features.fast_leaf_culling.MixinLeavesBlock",
```

Comment:
This isn't about making leaf culling "fast", it's about fixing a bug. We should probably toss this and the other injection for RenderLayers into`features.render_layers.leaves` so that it can be disabled by config if problems arise (there are other mods which try to do this very thing...)